nnsvs/bin/conf/synthesis/config.yaml

```diff
@@ -39,8 +39,6 @@ timelag:
   in_scaler_path:
   out_scaler_path:
   model_yaml:
-  stream_sizes: [1]
```

Comment:
Seems like the config changes are not related to MDN? If so, it would be great if you split this into a separate PR, then I can merge it quickly.

Reply:
As you wrote, the config changes are not related to MDN directly.  I'll split this into another PR.